pontoon/base/models.py

```diff
@@ -2405,8 +2415,28 @@ def missing_without_unreviewed(self, locale, project=None):
         return ~Q(
             pk__in=self.get_filtered_entities(
                 locale,
-                Q(approved=True) | Q(fuzzy=True) | Q(rejected=False),
-                lambda x: x.approved or x.fuzzy or not x.rejected,
+                Q(approved=True) | Q(pretranslated=True) | Q(rejected=False),
+                lambda x: x.approved or x.pretranslated or not x.rejected,
+                project=project,
+            )
+        )
+
+    def fuzzy(self, locale, project=None):
+        """Return a filter to be used to select entities marked as "fuzzy".
```

Comment:
We're keeping the Fuzzy filter around and moving it from Status to Extra filters in case users want to list them separately from other Missing strings.

As for the Missing filter, we're applying the change actually - it catches Fuzzy strings now.